Document/0x05d-Testing-Data-Storage.md

```diff
@@ -311,21 +311,21 @@ The typical example of Android Keystore attestation response looks like this:
 ```
 
 In the above JSON snippet, the keys have the following meaning:
-        `fmt`: Attestation statement format identifier
-        `authData`: It denotes the authenticator data for the attestation
-        `alg`: The algorithm that is used for the Signature
-        `sig`: Signature
-        `x5c`: Attestation certificate chain
+- `fmt`: Attestation statement format identifier
+- `authData`: It denotes the authenticator data for the attestation
+- `alg`: The algorithm that is used for the Signature
+- `sig`: Signature
+- `x5c`: Attestation certificate chain
 
-Note: The `sig` is generated by concatenating `authData` and `clientDataHash` (challenge sent by the server) and signing through the credential private key using the `alg` signing algorithm and the same is verified at the server-side by using the public key in the first certificate.
+> Note: The `sig` is generated by concatenating `authData` and `clientDataHash` (challenge sent by the server) and signing through the credential private key using the `alg` signing algorithm. The same is verified at the server-side by using the public key in the first certificate.
 
-For more understanding on the implementation guidelines, [Google Sample Code](https://github.com/googlesamples/android-key-attestation/blob/master/server/src/main/java/com/android/example/KeyAttestationExample.java "Google Sample Code For Android Key Attestation") can be referred.
+For more understanding on the implementation guidelines, you can refer to [Google Sample Code](https://github.com/googlesamples/android-key-attestation/blob/master/server/src/main/java/com/android/example/KeyAttestationExample.java "Google Sample Code For Android Key Attestation").
 
-For the security analysis perspective the analysts may perform the following checks for the secure implementation of Key Attestation:
+For the security analysis perspective, the analysts may perform the following checks for the secure implementation of Key Attestation:
 
-- Check if the key attestation is totally implemented at the client-side. In such scenario, the same can be easily bypassed by tampering the application, method hooking, etc.
+- Check if the key attestation is totally implemented on the client-side. In such scenario, the same can be easily bypassed by tampering the application, method hooking, etc.
```

Comment:
What do you want to say here??
"In such scenario, the same can be easily bypassed by tampering the application, method hooking, etc."

Reply:
I agree that this is not clear at all. I would assume that whoever wow this was trying to say something like "it's important to check whether the key attestation is implemented entirely on the client side (i.e., within the application itself), because if it is, then it can be more easily bypassed by attackers".

Reply:
Thanks @cpholguera. Let's have a go at it then.

Reply:
Feel free to change it more. Maybe use some text from my comment. I feel the current version is still missing something :/